packages/gdb.rb

```diff
@@ -6,18 +6,18 @@
 class Gdb < Autotools
   description 'The GNU Debugger'
   homepage 'https://www.gnu.org/software/gdb/'
-  version "15.1-#{CREW_GCC_VER}-#{CREW_PY_VER}"
+  version "16.1-#{CREW_GCC_VER}-#{CREW_PY_VER}"
   license 'GPL3'
   compatibility 'all'
-  source_url "https://ftpmirror.gnu.org/gnu/gdb/gdb-#{version.split('-').first}.tar.xz"
+  source_url "https://ftp.gnu.org/gnu/gdb/gdb-#{version.split('-').first}.tar.xz"
```

Comment:
The ftpmirror url picks the closest mirror site which can increase efficiency.

Reply:
Yes, but think about how few people build from source?

Also, sources at mirrors were unavailable...

Reply:
I have never noticed a down mirror site yet but I definitely believe it's possible.